prov/rxm/src/rxm_conn.c

```diff
@@ -599,16 +639,37 @@ int rxm_cmap_process_connreq(struct rxm_cmap *cmap, void *addr,
 	return ret;
 }
 
+int rxm_msg_eq_progress(struct rxm_ep *rxm_ep)
+{
+	struct rxm_msg_eq_entry *entry;
+       int ret;
+
+	entry = alloca(RXM_MSG_EQ_ENTRY_SZ);
+	if (!entry) {
+		FI_WARN(&rxm_prov, FI_LOG_EP_CTRL,
+			"unable to allocate memory!\n");
+		return -FI_ENOMEM;
+	}
+
+	while (1) {
+		memset(entry, 0, RXM_MSG_EQ_ENTRY_SZ);
```

Comment:
Is the memset needed inside the loop?

Reply:
I think at least the data field of fi_eq_cm_entry needs to be cleared if not the others. This is needed to correctly process FI_CONNECTED event.

Reply:
It seems that may not be necessary as we have a check for entry->rd before accessing cm_data. I'll remove this.